ibm/mas_devops/roles/appconnect/defaults/main.yml

```diff
@@ -1,15 +1,13 @@
 ---
-# MAS 8.4 uses AppConnect channel subscription v1.4 and license id = L-APEH-BSVCHU
-# MAS 8.5 uses AppConnect channel subscription v2.0 and license id = L-KSBM-BZWEAT
 # MAS 8.6 uses AppConnect channel subscription v2.0 and license id = L-KSBM-C37J2R
 # MAS 8.7 uses AppConnect channel subscription v3.0 and license id = L-KSBM-C87FU2
+# MAS 8.8 uses AppConnect channel subscription v4.2 and license id = L-APEH-C9NCK6
 
-# Define where AppConnect will be installed
+# Define where AppConnect will be installed, the default values below
 appconnect_namespace: "{{ lookup('env', 'APPCONNECT_NAMESPACE') | default('ibm-app-connect', true) }}"
-# v3.0 channel is used in MAS 8.7, v2.0 channel is used in older MAS versions
-appconnect_channel: "{{ lookup('env', 'APPCONNECT_CHANNEL') | default('v3.0', true) }}"
-appconnect_dashboard_name: "{{ lookup('env', 'APPCONNECT_DASHBOARD_NAME') | default('dashboard-12020r2', true) }}"
-appconnect_license_id: "{{ lookup('env', 'APPCONNECT_DASHBOARD_NAME') | default('L-KSBM-C87FU2', true) }}"
+appconnect_channel: "{{ lookup('env', 'APPCONNECT_CHANNEL') | default('v4.2', true) }}"
+appconnect_dashboard_name: "{{ lookup('env', 'APPCONNECT_DASHBOARD_NAME') | default('dashboard-12040r2', true) }}"
+appconnect_license_id: "{{ lookup('env', 'APPCONNECT_DASHBOARD_NAME') | default('L-APEH-C9NCK6', true) }}"
```

Comment:
This should be APPCONNECT_LICENSE_ID env var.

Reply:
oh that's true... glad we just go if the default most of the times, this was broken for quite some time... just fixed.